internal/pay/data.go

```diff
@@ -63,23 +67,33 @@ type ThreeDSecure struct {
 }
 
 type RefundSummary struct {
-	Status          string `json:"status"`
-	AmountAvailable int    `json:"amount_available"`
+	Status          string      `json:"status"`
+	AmountAvailable AmountPence `json:"amount_available"`
 }
 
 type SettlementSummary struct {
-	CaptureSubmitTime string `json:"capture_submit_time"`
-	CapturedDate      string `json:"captured_date"`
-	SettledDate       string `json:"settled_date"`
+	CaptureSubmitTime time.Time `json:"capture_submit_time"`
+	CapturedDate      date.Date `json:"captured_date"`
+	SettledDate       date.Date `json:"settled_date"`
+}
+
+type AmountPence int
+
+func (a AmountPence) Pounds() string {
+	return fmt.Sprintf("£%s", humanize.CommafWithDigits(float64(a)/100, 2))
```

Comment:
I've started trying to avoid `fmt.Sprintf` if it can be replaced by `"x" + y`, as I think it looks less clear in these simple cases (plus premature optimisation 💪)

Reply:
Old habits die hard - I'd got into this in PHP ages ago but agree its generally more readable with simple strings.